app/logic/Abstract/Account.ts

```diff
@@ -116,6 +116,7 @@ function findFreeAccountID(): string {
 function getAllAccounts(): Collection<Account> {
   let allAccounts = new ArrayColl<Account>();
   allAccounts.addAll(appGlobal.emailAccounts);
+  allAccounts.addAll(appGlobal.emailAccounts.map(acc => acc.outgoing));
```

Comment:
Note that not all email accounts have outgoing accounts associated with them.

Reply:
Thanks for the heads up.